.ci/dev/publish-branch/Preview.Jenkinsfile

```
@@ -0,0 +1,62 @@
+@Library('corda-shared-build-pipeline-steps')
+import static com.r3.build.BuildControl.killAllExistingBuildsForJob
+
+killAllExistingBuildsForJob(env.JOB_NAME, env.BUILD_NUMBER.toInteger())
+
+pipeline {
+    agent { label 'k8s' }
+
+    options {
+        timestamps()
+        ansiColor('xterm')
+        overrideIndexTriggers(false)
+        buildDiscarder(logRotator(daysToKeepStr: '7', artifactDaysToKeepStr: '7'))
+        timeout(time: 3, unit: 'HOURS')
+    }
+
+    environment {
+        ARTIFACTORY_BUILD_NAME = "Corda / Publish / Publish Preview to Artifactory"
```

see a comment for the other file
```suggestion
        ARTIFACTORY_BUILD_NAME = "Corda :: Publish :: Publish Preview to Artifactory"
```